executor/src/witgen/jit/block_machine_processor.rs

```diff
@@ -399,7 +439,7 @@ params[2] = Add::c[0];"
     }
 
     #[test]
-    #[should_panic = "Column NotStackable::a is not stackable in a 1-row block"]
+    // TODO #[should_panic = "Column NotStackable::a is not stackable in a 1-row block"]
```

Comment:
This does not fail any more because it just removes the assignment that comes from evaluating `a = a'`. This is kind of consistent with our termination condition of "all outputs are known, all machine calls are made".

I think we have to re-think both the termination condition of a block machine and also maybe do better shape detection.

But maybe not as part of this PR?